### DISCUSSION TITLE
MT-GPR

### DISCUSSION THREAD

**Discussion by AdrianSosic** [emdgroup/baybe#80] [Transfer Learning]
paper: https://www.sciencedirect.com/science/article/pii/S0927025622005705?via%3Dihub

- can it generalize to multiple tasks
- in which bucket of the taxonomy does it fall
- how does it relate to the other algorithms in that bucket
- what are the advantages / disadvantages
- has the paper been applied / cited?

**Comment by Scienfitz** [comment]
closing as no longer pursued
